Misc/NEWS

```diff
@@ -281,6 +281,12 @@ Extension Modules
 Library
 -------
 
+- bpo-29842: Executor.map no longer creates all futures eagerly prior to
+  yielding any results. This allows it to work with huge or infinite iterables
+  without consuming excessive resources or crashing, making it more suitable
+  as a drop in replacement for the built-in map.
+  Patch by Josh Rosenberg.
+
```

Comment:
Would you add NEWS entry file using blurb?

Reply:
My understanding is that NEWS is now maintained automatically, so any such change is no longer necessary.

Reply:
Hmm... Apparently it's a new system, but not automatic. Trying to figure that out now.

Reply:
Thanks to Gregory Smith, I now know about the blurb tool (didn't realize you meant "blurb" as a tool name, sorry), and have used it. Entry now exists.